README.md

```diff
@@ -533,6 +430,135 @@ CirrusMDSDK.singleton.onRemoteNotificationSelected(streamId: String, eventType:
+
+[CirrusMDSDK.singleton setConfig:config];
+```
+
+### Titles and Colors
+
+Some of the navigation titles in the SDK are configurable. To override the default set a title on your `CirrusMDSDKConfig` object.
+Many of the colors in the SDK are configurable. To override the colors set them in on your `CirrusMDSDKConfig` object.
+
+```swift
+primary     // defaults to "#1a1a1a", used in avatar outlines and buttons
+offHours    // defaults to "#4a4a4a", used in the Patient Encounter Queue
+unassigned  // defaults to "#0c4c78", used in the Patient Encounter Queue
+assigned    // defaults to "#44db5e", used in the Patient Encounter Queue
+error       // defaults to "#dd0000", will be used in the future
+success     // defaults to "#44db5e", will be used in the future
+warning     // defaults to "#daaf0f", used in the offline banner
+```
+
+##### Swift
+
+```swift
+let config = CirrusMDSDKConfig()
+config.title = "Custom Title Here" // defaults to "My Healthcare Services" if this is not set
+config.primary = "FF5733"
+
+CirrusMDSDK.singleton.setConfig(config)
+```
+
+##### Obective-C
+
+```obj-c
+CirrusMDSDKConfig* config = [[CirrusMDSDKConfig alloc] init];
+config.title = @"Custom Title Here"; // defaults to "My Healthcare Services" if this is not set
+config.primary = @"FF5733";
+
+[CirrusMDSDK.singleton setConfig:config];
+```
+
+### Enable Settings View
+
+There is an optional Settings view that you can allow your users to have access to. The Settings view, when enabled, is accessed via a button in the navigation bar of the SDK. This Settings view allows the user to view and edit their profile, medical history, dependents, permissions, and Terms of Use / Privacy Policy. The Settings view also allows the user to manually log out of the CirrusMDSDK. The availability of the Settings view is controlled by the `CirrusMDSDKConfig`.
+
+NOTE: The Settings view defaults to be disabled.
+
+##### Swift
+
+```swift
+let config = CirrusMDSDKConfig()
+config.enableSettings = true
+
+CirrusMDSDK.singleton.setConfig(config)
+```
+
+##### Obective-C
+
+```obj-c
+CirrusMDSDKConfig* config = [[CirrusMDSDKConfig alloc] init];
+config.enableSettings = YES;
+
+[CirrusMDSDK.singleton setConfig:config];
+```
+
+### Enable Dependents
+
+The CirrusMDSDK can support a user having dependents that can chat under their guarantor's account. When dependents support is enabled and a user has dependents they will see a dependents button that allows them to switch to chatting as that dependent. Support for dependents is controlled by the `CirrusMDSDKConfig`.
+
+Note: Dependent support defaults to being disabled.
+
+##### Swift
+
+```swift
+let config = CirrusMDSDKConfig()
+config.enableDependents = true
+
+CirrusMDSDK.singleton.setConfig(config)
+```
+
+##### Obective-C
+
+```obj-c
+CirrusMDSDKConfig* config = [[CirrusMDSDKConfig alloc] init];
+config.enableDependents = YES;
+
+[CirrusMDSDK.singleton setConfig:config];
+```
+
+### Enable User Log Out
```

Comment:
Moved some documentation around so the README diff isn't great. This is documentation for the new enableUserLogOut config option.